src/include/catalog/column_catalog.h

```diff
@@ -47,6 +47,7 @@ class ColumnCatalogObject {
   inline uint32_t GetColumnId() { return column_id; }
   inline uint32_t GetColumnOffset() { return column_offset; }
   inline type::TypeId GetColumnType() { return column_type; }
+  inline size_t GetColumnLength() { return column_length; }
```

Comment:
Would `unit32_t` same as `GetColumnOffset()` be sufficient?

Reply:
The column length in `storage/column.h` is `size_t`.  I took same type, but I can't  decide which is better.

Reply:
Could you please change the type of `GetColumnOffset()` to `oid_t` then?